public/icons/docs/vuex/SOURCE

```diff
@@ -0,0 +1 @@
+http://vuejs.org/
```

Comment:
should this be https://vuex.vuejs.org/

Reply:
I’ve used the exact same icon than in the Vue.js documentation set, so it should probably be the same.

Reply:
I see, thanks.